cmd/cpu-device-plugin/cpu-device-plugin.go

```diff
@@ -32,6 +34,15 @@ type cpuDeviceManager struct {
 	poolType       string
 }
 
+type nodeCpuTopology struct {
+	cpuList []cpu
+}
+
+type cpu struct {
+	coreID int
+	socketID int
```

Comment:
I used the term socket all over the code because some Intel folks were saying in the past the two are not always the same, and socket is the more accurate term
But I'm no HW expert, so if you tell me this is actually NUMA node I'm totally fine to rename :)

Reply:
k, considering lscpu man says it is nodeID, I renamed the stuff accordingly!

Reply:
Yeah, I think target is to align device, memory (hugepage) and cpu allocation to same NUMA node so we are talking about NUMA node here.

Reply:
okay, changed the name